src/Jobs/Recruit.Vacancies.Jobs/DomainEvents/Handlers/Vacancy/DraftVacancyUpdatedHandler.cs

```diff
@@ -24,9 +24,11 @@ public async Task HandleAsync(string eventPayload)
             try
             {
                 _logger.LogInformation($"Processing {nameof(DraftVacancyUpdatedEvent)} for vacancy: {{VacancyId}}", @event.VacancyId);
-               
+
+                await _client.EnsureVacancyHasReferenceNumberAsync(@event.VacancyId);
```

Comment:
Are they awaited separately to avoid the vacancy overwrite issue?

Reply:
Yes that was my thinking.

Reply:
👍